web/src/pages/deevan.tsx

```diff
@@ -0,0 +1,37 @@
+import Helmet from 'react-helmet';
+
+import Page from '../components/Page';
+import Section, { SectionTitle } from '../components/Section';
+import { TopNavigation } from '../components/Navigation';
+import Footer from '../components/Footer';
+import env from '../env/env';
+import Link from '../components/Link';
+import { useReq } from '../helpers/networking';
+
+const PrivacyPage = () => {
```

Comment:
consider: Rename this to DeevanPage so that in the React debugging tools it's clear where errors originate etc.